Solution1/Solution1.Tests/BL/Services/WeatherApiServiceTests.cs

```diff
@@ -56,5 +58,83 @@ public async Task GetByCityNameAsync_ReturnedWeatherApiDTO_Success()
             var expectedWeatherApiDto = new WeatherApiDTO() { CityName = cityName, TemperatureValues = new WeatherApiTempDTO() { Temp = 1.86 } };
             Assert.True(new CompareLogic().Compare(expectedWeatherApiDto, result).AreEqual);
         }
+
+        [Fact]
+        public async Task GetForecastByCityNameAsync_ReturnedForecastWeatherApiDTO_Success()
+        {
+            // Arrange
+            var cityName = "Minsk";
+            var lat = 53;
+            var lon = 27;
+            var urlCoordinates = $"http://api.openweathermap.org/geo/1.0/direct?q={cityName}&appid=3fe39edadae3ae57d133a80598d5b120";
+            var urlForecast = $"https://api.openweathermap.org/data/2.5/forecast?lat={lat}&lon={lon}&cnt=2&units=metric&appid=3fe39edadae3ae57d133a80598d5b120";
+            
+            var responseCoordinates = new HttpResponseMessage
+            {
+                StatusCode = HttpStatusCode.OK,
+                Content = new StringContent(@"[{""name"": ""Minsk"", ""lat"" : 53, ""lon"" : 27}]"),
```

Comment:
don't hardcode JSON. Use Json Serializer and anonymous objects

Reply:
+